docs/firmware/zephyr-device-sdk/authentication/certificate-auth.md

```diff
@@ -65,9 +65,9 @@ converted to the `DER` binary format for use with the Golioth Zephyr SDK.
 
 ```shell
 PROJECT_SLUG='your-golioth-projectID'
-PRIMARY_HARDWARE_ID='choose-hardware-id-for-this-device'
+DEVICE_NAME='choose-a-name-for-this-device'
```

Comment:
```suggestion
DEVICE_NAME='choose-a-unique-name-for-this-device'
```

I'm going to force push this small change. I think it's important to indicate that a unique device name should be used for each cert generated.